src/main/java/org/jabref/gui/preferences/journals/JournalAbbreviationsTab.java

```diff
@@ -132,11 +161,31 @@ private void removeList() {
     @FXML
     private void addAbbreviation() {
         viewModel.addAbbreviation();
-        searchBox.textProperty().set("");
+        if (!searchBox.getText().isEmpty()) {
+            invalidateSearch.play();
+        }
         selectNewAbbreviation();
         editAbbreviation();
     }
 
+    private static StringProperty createFlashingColorStringProperty(final ObjectProperty<Color> flashingColor) {
+        final StringProperty flashingColorStringProperty = new SimpleStringProperty();
+        setColorStringFromColor(flashingColorStringProperty, flashingColor);
+        flashingColor.addListener((observable, oldValue, newValue) -> setColorStringFromColor(flashingColorStringProperty, flashingColor));
+        return flashingColorStringProperty;
+    }
+
+    private static void setColorStringFromColor(StringProperty colorStringProperty, ObjectProperty<Color> color) {
+        colorStringProperty.set(
```

Comment:
Use ColorUtil.ToRGB()  from org.jabref.gui.util;

Reply:
That does not include opacity. I could change it to include the opacity, but wouldn't know how to start making sure that nothing else breaks.
Shall I move the method 'setColorStringFromColor' I created to ColorUtil and rename it to 'toRGBACode'?

Reply:
Yeah, just add a second there

Reply:
Done. Also added some tests for the new ColorUtil Method. Don't know how much benefit there is in having them, as some of the string needs to be made-up by String.format so they partially do the same thing the method does. This is so the floating point numbers are represented the same, since I think they may be different on other systems (precision-wise).